plugins/module_utils/network/iosxr/config/lag_interfaces/lag_interfaces.py

```diff
@@ -156,7 +156,10 @@ def set_state(self, want, have):
         state = self._module.params["state"]
         commands = []
 
-        if self.state in ("merged", "replaced", "rendered") and not want:
+        if (
+            state in ("overridden", "merged", "replaced", "rendered")
+            and not want
+        ):
```

Comment:
I don't see what this change has to do with the rest of this PR, and including it would complicate a clean backport to 2.9

Reply:
I tried to fix missing check for "overriden" state in a93b8c54dc4f730d03206d1ee57d512ea1515ac5
because it broke the tests, but this commit doesn't directly related to issues described in PR

If it is necessary i can revert this commit.

Reply:
reverted this and merge with master